packages/edit-post/src/editor.js

```diff
@@ -31,14 +31,28 @@ function Editor( {
 	initialEdits,
 	...props
 } ) {
+	// Get the rendering mode for the post type.
+	const postTypeRenderingMode = useSelect(
```

Comment:
Do we want to do the same in the site editor too? We can actually unify this in the "editor" package as well.

Worth-noting though that if we decide to unify, we'll have to make a decision about the "page" post type. Right now it's default rendering is "post-only" in the post editor but "template-locked" in the site editor.